services/restic_backup.md

```diff
@@ -115,6 +115,34 @@ For mariadb do the following:
 
 ## Restore from a backup
 
+To restore from a GUI 
+
+  1. Install the restic-browser app with 
+
+    ```bash
+    brew install restic-browser
+    ```
+
+  2. Launch the application
+
+     Create a new Location:
+
+       * In the **Type**: Select Google Cloud Storage
+       * In the **Bucket**: Enter the name and path of your bucket. (e.g., gs:postgres-15-backup:daily)
+       * In the **GOOGLE_PROJECT_ID**: Enter pul-gcdc
+       * In the **GOOGLE_APPLICATION_CREDENTIALS**: Get a copy of the credentials and place them at a know location
+         * Log into `lib-postgres-prod1` and download or copy the credentials with the following
+           ```bash
+           sudo su - postgres
+           cat ~/.restic/pul-gcdc-UUIDnumber.json
+           ```
+       * In the **Repository Password**: Get the contents from the server
+         * Log into `lib-postgres-prod1` and get the password with the following
+           ```bash
+           sudo su - postgres
+           cat ~/.restic.pwd
+           ```
+
 To restore the latest usable postgresql backup from restic, run the following commands:
 
   1. As the postgresql user run the following steps:
```

Comment:
Is this meant to be run one one of the postgres servers? It would be convenient to run locally to get fresh data for development work. I tried locally and got an error until I set GOOGLE_APPLICATION_CREDENTIALS in my environment but I must have syntax wrong because it's complaining about a storage object not existing.   I tried setting `RESTIC_REPOSITORY` to the location I just created in restic browser but I'm not sure if that's correct.

Could you use an actual path as an example here so I can more easily check if my setup is working?

Or is it better if I try running this on one of the staging postgres servers?